consul/consul.go

```diff
@@ -1,97 +1,52 @@
 package consul
 
 import (
+	"github.com/alaa/pencil-go/registry"
```

Comment:
As I know Jason was talking with Alaa about that and they decided that it stays within Alaa account